client/tailwind.config.js

```diff
@@ -109,18 +111,15 @@ module.exports = {
         "light-surface-variant": "#F3DDDF",
         "light-on-surface-variant": "#524345",
         "light-outline": "#847374",
-        "state-layers": {
-          "light-primary": "#6750A4",
-          "on-secondary-container": "#1D192B",
-          "light-on-surface": "#1C1B1F",
-        }
-      },
+        "text-secondary": "#380037",
```

Comment:
Is this specified anywhere in the [color palette](https://www.figma.com/file/PHXIBriEHdNfYd4WXXTDJx/Material-Theme-Builder-(Community)?node-id=0%3A1&t=ZiIFGt7SKi0Or90R-3)? I could miss it...lmk